configs/mmpose/yolox-pose_onnxruntime_static.py

```diff
@@ -0,0 +1,22 @@
+_base_ = ['./pose-detection_static.py', '../_base_/backends/onnxruntime.py']
```

Comment:
could you test with dynamic batch, see config https://github.com/open-mmlab/mmdeploy/blob/main/configs/mmpose/pose-detection_simcc_onnxruntime_dynamic.py

Reply:
The `multiclass_nms` does not support outputing selected indices for dynamic batch.